controllers/activemqartemisaddress_controller_test.go

```diff
@@ -202,8 +195,126 @@ var _ = Describe("Address controller", func() {
 			}
 		})
 	})
+
+	Context("Address delete and scale down", func() {
```

Comment:
@howardgao I started trying to reproduce the issue with deletion when there is a consumer from the the cluster connection, but this test with a cluster in play works fine on main. It did surface a panic when there was no queue and just an address in the CR, but worked otherwise.